libs/ui/design-system/src/lib/tabs.tsx

```diff
@@ -0,0 +1,27 @@
+import * as TabsPrimitive from '@nmit-coursition/ui/primitives/tabs'
+import type { ReactNode } from 'react'
+
+interface TabsProps<Values extends string[]> {
+  triggers: { value: NoInfer<Values[number]>; displayText: string }[]
```

Comment:
**question:** Is it possible to have different length, therefore breaking the component? If so, we should alter the API.

Reply:
Nicee!!
You overly simplified it.